angular.json

```diff
@@ -0,0 +1,124 @@
+            ],
+            "styles": [
+              "src/styles.css"
+            ],
+            "scripts": []
+          },
+          "configurations": {
+            "production": {
+              "fileReplacements": [
+                {
+                  "replace": "src/environments/environment.ts",
+                  "with": "src/environments/environment.prod.ts"
+                }
+              ],
+              "optimization": true,
+              "outputHashing": "all",
+              "sourceMap": false,
+              "extractCss": true,
+              "namedChunks": false,
+              "extractLicenses": true,
+              "vendorChunk": false,
+              "buildOptimizer": true,
+              "budgets": [
+                {
+                  "type": "initial",
+                  "maximumWarning": "2mb",
+                  "maximumError": "5mb"
+                },
+                {
+                  "type": "anyComponentStyle",
+                  "maximumWarning": "6kb",
+                  "maximumError": "10kb"
+                }
+              ]
+            }
+          }
+        },
+        "serve": {
+          "builder": "@angular-devkit/build-angular:dev-server",
+          "options": {
+            "browserTarget": "ShutterFeed:build"
+          },
+          "configurations": {
+            "production": {
+              "browserTarget": "ShutterFeed:build:production"
+            }
+          }
+        },
+        "extract-i18n": {
+          "builder": "@angular-devkit/build-angular:extract-i18n",
+          "options": {
+            "browserTarget": "ShutterFeed:build"
+          }
+        },
+        "test": {
+          "builder": "@angular-devkit/build-angular:karma",
+          "options": {
+            "main": "src/test.ts",
+            "polyfills": "src/polyfills.ts",
+            "tsConfig": "tsconfig.spec.json",
+            "karmaConfig": "karma.conf.js",
+            "assets": [
+              "src/favicon.ico",
+              "src/assets"
+            ],
+            "styles": [
+              "src/styles.css"
+            ],
+            "scripts": []
+          }
+        },
+        "lint": {
+          "builder": "@angular-devkit/build-angular:tslint",
+          "options": {
+            "tsConfig": [
+              "tsconfig.app.json",
+              "tsconfig.spec.json",
+              "e2e/tsconfig.json"
+            ],
+            "exclude": [
+              "**/node_modules/**"
+            ]
+          }
+        },
+        "e2e": {
+          "builder": "@angular-devkit/build-angular:protractor",
+          "options": {
+            "protractorConfig": "e2e/protractor.conf.js",
+            "devServerTarget": "ShutterFeed:serve"
+          },
+          "configurations": {
+            "production": {
+              "devServerTarget": "ShutterFeed:serve:production"
+            }
+          }
+        }
+      }
+    }},
+  "defaultProject": "ShutterFeed"
```

Comment:
You created this project leaving CSS as default styling, but we could do much more with SCSS.

SCSS is a superset of CSS that uses the same syntax and has a lot more features than CSS, such as nesting, variables, loops, etc.

Here's an example of what you can do in SCSS: 
```scss
:host { //host tells the angular to only apply this style to the component that requests this stylesheet
  h1 {
    color: $primary-color;
  }
  
  a {
    font-decoration: none;
    &:hover {
        color: $accent-color;
    }
  }
}
```
Which in vanilla CSS would look like:
```css
:host h1 {
  color: red;
}
  
:host a {
    font-decoration: none;
}
:host a:hover {
  color: yellow;
}
```

SCSS is can become very useful when working on much larger stylesheets, and the Material library heavily relies on SCSS for styling, so we should switch the project to SCSS simply by appending this after this line:
```javascript
"schematics": {
    "@schematics/angular:component": {
      "styleext": "scss"
    }
  }
```